src/Chapter01/Listing01.01.HelloWorldInC#.ps1

```diff
@@ -22,21 +23,24 @@ try {
     Set-Location ./HelloWorld/
     dotnet new console
     Get-Content Program.cs
-    dotnet run
     $codeListing = 
     @"
 class HelloWorld
 {
     static void Main()
     {
-        System.Console.WriteLine("Hello. My name is Inigo Montoya.");
+        Console.WriteLine("Hello. My name is Inigo Montoya.");
     }
 }
 "@
 
     $codeListing > Program.cs
     Get-Content Program.cs
-    dotnet run
+
+    Write-Warning "Remove dotnet build step once we upgrade to version 10"
+    # Build with version 10 because dotnet version 6 defaults to an ImplicitUsings element in the project file.
+    dotnet build /property:LangVersion=10
+    dotnet run --no-build
```

Comment:
```suggestion
    dotnet run
```

Aren't we in v11 in this pr now? Should this be updated to remove the build step and remove the warning?

Reply:
Agreed!